include/zephyr/bluetooth/classic/classic.h

```diff
@@ -38,10 +38,20 @@ extern "C" {
  * @{
  */
 
+/** @brief BR/EDR discovery private structure */
```

Comment:
Either drop the doxygen annotation from the comments, or mark this as hidden or private (there's dedicated oxygen syntax for it, IIRC)